src/olympia/pages/templates/pages/shield_study_11.html

```diff
@@ -2,58 +2,176 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-
 <html lang="en">
```

Comment:
This is probably wrong (best to leave it out, or maybe `{{ request.LANG }}` would work.